lib/components/button/button.tsx

```diff
@@ -51,12 +53,48 @@ export const Button: ButtonComponent = forwardRef(function Button(
       aria-disabled={disabled}
       {...props}
     >
+      {variant === "add" && !isLoading && <AddIcon />}
       {isLoading && (
         <span className="button-loading-indicator">
           <span>Loading...</span>
         </span>
       )}
       {isLoading && loadingText ? loadingText : children}
+      {variant === "chevron" && <ChevronIcon />}
     </ButtonComp>
   );
 });
+
+const AddIcon = () => {
```

Comment:
This will throw an error with the new eslint config, `no use before defined`.
Either move the icons above the component, or separate them from the file.

Reply:
Also I wonder if we should hide the right chevron when its loading?

Reply:
OK, I'll shift them up, thanks.

I had a preview of how the icon looked with the spinner and chevron and thought it looked fine.

Reply:
ok then lets leave the last comment. thanks!